code/game/objects/items/clown_items.dm

```diff
@@ -126,6 +126,7 @@
 				REMOVE_TRAIT(target, TRAIT_SPRAYPAINTED, CRAYON_TRAIT)
 			target.wash(CLEAN_SCRUB)
 			target.remove_atom_colour(WASHABLE_COLOUR_PRIORITY)
+			addfingerprint("Traces of Soap") // Yay my first PR ! This makes it so soap leaves a trace
 			decreaseUses(user)
```

Comment:
seems like you accidentally added this to your PR?

Reply:
I did, I dont know why it still thought I was on traces of soap